### DISCUSSION TITLE
Error code 3 on print to 3d

### DISCUSSION THREAD

**Discussion by brooked3d** [p3d-dev/Print-To-3D#18] [General]
What is the error code 3? I have been getting the same error code on all my projects and I don’t know what it is or what settings to change to correct it. Does anyone know what the error code is?

**Comment by p3d-dev** [comment]
Sorry to hear, that you encounter a problem with PrintTo3D. In order to gives us some context, we would appreciate your support. Could you please provide a screenshot and describe, when exactly this error code pops up ?

**Comment by brooked3d** [comment]
It’s been popping up every time I try to slice with multiple projects


Sent from Yahoo Mail for iPhone


On Sunday, April 21, 2024, 9:46 AM, p3d-dev ***@***.***> wrote:




Sorry to hear, that you encounter a problem with PrintTo3D. In order to gives us some context, we would appreciate your support. Could you please provide a screenshot and describe, when exactly this error code pops up ?

**Reply by p3d-dev** [reply]
Any chance to provide a screenshot of the error message ?

**Reply by brooked3d** [reply]
![IMG_0261](https://github.com/p3d-dev/Print-To-3D/assets/167690771/2939caec-4a9f-44c9-83b6-5909bf4fc743)

**Reply by p3d-dev** [reply]
Thanks a lot. Now it is clear and we can check for the root cause.

**Reply by brooked3d** [reply]
Has the issue been resolved?

**Reply by brooked3d** [reply]
Has the issue been resolved ? Or can you tell me what i need to do to get it to work?

**Reply by p3d-dev** [reply]
Unfortunately, the issue has not been resolved. From our investigation, the slicer process has been terminated and this has happened without further information about possible root cause.

Our best guess is, that the slicer process termination may be due to some printer/process configuration settings, which unexpectedly yields this exception.

Remedy: Start from scratch with new clean printer/process profiles. Default profiles can be created via Sidebar by long press on the "+" symbol and then select "Standard" in the popup menu.

In case this is fixing it, please send us those settings causing this issue - Using Share button/Save File. This should allow us to reproduce the issue on our devices, then narrow down on the problematic settings and inform you about needed changes in your user configuration and prevent it from happening again (or better error reporting) on app side for the next release.
